content/en/examples/policy/privileged-psp.yaml

```diff
@@ -3,7 +3,7 @@ kind: PodSecurityPolicy
 metadata:
   name: privileged
   annotations:
-    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+    seccomp.security.kubernetes.io/allowedProfileNames: '*'
```

Comment:
@hasheddan @saschagrunert The KEP is not explicit on this point, but I thought we were keeping the PSP seccomp feature in alpha as we were [keeping PSP functionality unchanged](https://github.com/kubernetes/enhancements/pull/1747#issuecomment-631105776). I am unsure whether it is worth getting users to "migrate" from the existing annotation to one without `alpha`.

Reply:
Yeah I think you are right. It does feel a little strange that this is `alpha` while seccomp itself is GA, but this can be changed in a future release if needed :+1:

Reply:
Adding @tallclair @liggitt who were involved in the discussions around the subject.